go/grpcweb/websocket_wrapper.go

```diff
@@ -31,6 +34,36 @@ func newWebSocketResponseWriter(wsConn *websocket.Conn) *webSocketResponseWriter
 	}
 }
 
+func (w *webSocketResponseWriter) EnablePing(timeOutInterval time.Duration) {
```

Comment:
Lets unexport this function, I don't think it needs to be exported.
```suggestion
func (w *webSocketResponseWriter) enablePing(timeOutInterval time.Duration) {
```